_posts/2022-09-29-r-text/r-text.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "The text-package"
```

Comment:
How about 

"Introducing the text package"

for a title?

Reply:
sounds great, thanks